submissions/alstep07/html-movie-seat-booking/script.js

```diff
@@ -0,0 +1,72 @@
+const seatsForm = document.querySelector('#tickets');
+const seatsCart = document.querySelector('#cart-tickets');
+const cartTotal = document.querySelector('#cart-total');
+const cartConfirmButton = document.querySelector('#cart-confirm');
+const cartCancelButton = document.querySelector('#cart-cancel');
+const modal = document.querySelector('#modal');
+const modalConfirmButton = modal.querySelector('#modal-confirm');
+const modalCancelButton = modal.querySelector('#modal-cancel');
+const modalTotal = modal.querySelector('#modal-total');
+const modalAmount = modal.querySelector('#modal-tickets');
+const PRICE = 5;
+
+const createSeatTemplate = (seat) => {
+  const template = `<div class="selected-seats__ticket">
+      <p class="ticket__location">${seat}</p>
+      <p class="ticket__price">$${PRICE.toFixed(2)}</p>
+    </div>`;
+  return template;
+};
+
+const createTotalTemplate = (seatsAmount) => {
+  const total = (PRICE * seatsAmount).toFixed(2);
+  const template = `Total: $${total}`;
+  return template;
```

Comment:
```suggestion
  return `Total: $${total}`;
```

No need to create a new variable.